### PR TITLE
AMQNET-588  Solution clean up

### PR DESCRIPTION
**Pull request by HavretGC**
- Reorganize repository structure
- Remove redundant .csproj files leaving only one csproj per actual project
- Update package scripts
- Replace sandcastle with docFX

### CODE REVIEW

**Comment by michaelandrepearce** [comment]
- [ ] tidy up commit so that moves don't appear as delete and add's thus keeping history
- [ ] make a doc csproj module, so that docfx builds as part of sln release build.

File: docs/template/partials/logo.tmpl.partial [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
```
Comment (michaelandrepearce) [inline docs/template/partials/logo.tmpl.partial]: please correct copyright.

**Comment by HavretGC** [comment]
Everything should be ok, now.

**Comment by michaelpearce-gain** [comment]
@HavretGC thanks for the help on this.

**Comment by michaelandrepearce** [comment]
@HavretGC thanks for the help on this.

**Pull request merged**